app/views/users/_header.html.erb

```diff
@@ -7,7 +7,9 @@
 <!--/descriptions-->
 
   <!--subnav-->
-  <%= render "users/header_navigation" %>
+  <% if logged_in? || policy(User).can_manage_users? %>
```

Comment:
I think this should check `logged_in_as_admin?` as well as the policy.

Reply:
Is there a case where the lack of explicit admin check here is causing an issue, like the navigation not showing when it should?

Reply:
Not as far as I'm aware. The motivation is mostly that admin_helper does it that way and that this code reads a bit confusing without it because the obvious not `logged_in?` case is a visitor, not an admin.

Reply:
I don't think it's worth making it more verbose if it's not necessary, then. Neither the comments helper nor the various views include that additional check, so I have a feeling the oddball check in admin helper might be either a relic from the very early days when we didn't include the admin check in `user_has_roles?(roles)` in the application policy, or maybe just a little leftover copypasta from previous methods in that file, which took admin as an argument, e.g.,

```
def admin_can_destroy_creations?(current_admin)
    return unless logged_in_as_admin?

    UserCreationPolicy.can_destroy_creations?(current_admin)
 end
```